app/Jobs/DispatchWebhooksJob.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace App\Jobs;
+
+use App\Events\Event;
+use App\Events\ShouldDispatchWebhooks;
+use App\Models\WebhookConfiguration;
+use Illuminate\Bus\Queueable;
+use Illuminate\Contracts\Queue\ShouldQueue;
+use Illuminate\Foundation\Bus\Dispatchable;
+use Illuminate\Queue\InteractsWithQueue;
+use Illuminate\Queue\SerializesModels;
+
+class DispatchWebhooksJob implements ShouldQueue
+{
+    use Dispatchable, InteractsWithQueue, Queueable, SerializesModels;
+
+    /**
+     * Create a new job instance.
+     */
+    public function __construct(private ShouldDispatchWebhooks&Event $event)
```

Comment:
```suggestion
    public function __construct(private ShouldDispatchWebhooks $event)
```

Wouldn't we only want the ones that have the interface?